src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/LanguageServices/LanguageServiceHandlerManager.cs

```diff
@@ -11,12 +11,18 @@ namespace Microsoft.VisualStudio.ProjectSystem.LanguageServices
     [Export]
     internal class LanguageServiceHandlerManager
     {
+        private readonly ICommandLineParserService _commandLineParser;
+
         [ImportingConstructor]
-        public LanguageServiceHandlerManager(UnconfiguredProject project)
+        public LanguageServiceHandlerManager(UnconfiguredProject project, ICommandLineParserService commandLineParser)
         {
             Requires.NotNull(project, nameof(project));
+            Requires.NotNull(commandLineParser, nameof(commandLineParser));
+
+            _commandLineParser = commandLineParser;
 
             EvaluationHandlers = new OrderPrecedenceImportCollection<IEvaluationHandler>(projectCapabilityCheckProvider: project);
+            CommandLineHandlers = new OrderPrecedenceImportCollection<ICommandLineHandler>(projectCapabilityCheckProvider: project);
```

Comment:
My first reaction to this was that it's a bit unsymmetric - for evaluation we have one handler per rule but for dt there's only rule and we have handler for processing components of that data flow. I would have expected an import of IDesignTimeHandlers and CommandLineHandler would have been the only export of it. This seems to bake in the fact that theres only one dt rule...